### DISCUSSION TITLE
While requesting the weaather.gov/alerts using lat= 45.47 and lon=9.19 getting json response as Invalid: out of bounds

### DISCUSSION THREAD

**Discussion by ChasingDeveloper** [weather-gov/api#519] [Q&A]
I'm requesting the weather.gov using the URL - https://api.weeather.gov/alerts/active?point=45.47,9.19 
I'm receiving the response as: 
{
   "correlationId": "20594c02",
   "title":  "Invalid Parameter",
   "type": "https://api.weather.gov/problems/InvalidParameter",
   "status": 400,
   "detail": "parameter \"point"\" is Invalid: out of bounds",
   "instance": "https://api.weather.gov/requests/20594c02"
}

as per the request the lat and lon are valid with the rules - lat ---> 90 to -90 and lon ----> 180 to -180 

Why the weather.gov is failing to send the response ? 

Can some here please assist and provide me the information. I appreciate it. Thanks!

@StephenClouse @scadergit

**Comment by jfbauer432** [comment]
I believe the lat/lon has to be within the united states.

**Reply by ChasingDeveloper** [reply]
@jfbauer432 

Okay. Can you please provide any reference link to this ? It's really hard to find out in weather.gov website. This helps me to complete my high priority work. 

Thanks!

**Comment by ktrue** [comment]
Your request for point 45.47,9.19 is outside of the USA (CONUS+Hawaii) .. The NOAA NWS forecasts are for the USA only.

**Reply by ChasingDeveloper** [reply]
@ktrue 

Okay. Can you please provide any reference link to this ? It's really hard to find out in weather.gov website. 

Thanks!

**Reply by ktrue** [reply]
I don't think it's posted anywhere, but it is widely known that the weather.gov site(s) provide weather information for the lower-48 USA states, Alaska, Hawaii, Puerto Rico, Guam only via the web services.  Yes, the GFS model is 'global', but the results for points outside what I'd described above are not provided by weather.gov (nor are expected to be provided).

If you want forecast information around the world, then you'll need to use a different API.  Both Aerisweather.com and weatherunderground.com provide such APIs (free for hobbyists submitting weather data to their respective sites, and for fee otherwise).
